src/store-credit/store-credit-reducer.ts

```diff
@@ -0,0 +1,57 @@
+import { combineReducers, composeReducers, Action } from '@bigcommerce/data-store';
+
+import { clearErrorReducer } from '../common/error';
+
+import { StoreCreditAction, StoreCreditActionType } from './store-credit-actions';
+import StoreCreditState, { StoreCreditErrorsState, StoreCreditStatusesState } from './store-credit-state';
+
+const DEFAULT_STATE: StoreCreditState = {
+    errors: {},
+    statuses: {},
+};
+
+export default function storeCreditReducer(
+    state: StoreCreditState = DEFAULT_STATE,
+    action: Action
+): StoreCreditState {
+    const reducer = combineReducers<StoreCreditState>({
+        errors: composeReducers(errorsReducer, clearErrorReducer),
+        statuses: statusesReducer,
+    });
+
+    return reducer(state, action);
+}
+
+function errorsReducer(
+    errors: StoreCreditErrorsState = {},
```

Comment:
Can you provide the default value defined in `DEFAULT_STATE`? i.e.: `DEFAULT_STATE.errors`.